component/alertrules.jsonnet

```diff
@@ -6,11 +6,10 @@ local params = inv.parameters.topolvm;
 
 local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
 
-assert
-  std.member(inv.applications, 'rancher-monitoring') ||
-  std.member(inv.applications, 'openshift4-monitoring') ||
-  std.member(inv.applications, 'prometheus')
-  : 'Neither rancher-monitoring nor openshift4-monitoring nor prometheus is available';
+// assert
+//   std.member(inv.applications, 'openshift4-monitoring') ||
+//   std.member(inv.applications, 'rancher-monitoring')
+//   : 'Neither rancher-monitoring nor openshift4-monitoring is available';
```

Comment:
If you want to remove stuff, remove it, don't just comment it out. However, I don't get why we want to remove this, if anything you'd just want to drop component rancher-monitoring and keep the check for either openshift4-monitoring or prometheus.

Reply:
Seems like an error happend there on my side, that was not the intention 😅

Reply:
The assert rule has to go, it won't compile using component-prometheus.

Reply:
> The assert rule has to go, it won't compile using component-prometheus.

Then we should fix it to compile with component-prometheus, or reduce the assertion to `openshift4-monitoring`.